<ui version="4.0" >
 <class>HttpRequestWindow</class>
 <widget class="QMainWindow" name="HttpRequestWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>274</width>
    <height>390</height>
   </rect>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <layout class="QGridLayout" >
    <item row="0" column="0" >
     <widget class="QLabel" name="label" >
      <property name="text" >
       <string>Wengo Login:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" colspan="2" >
     <widget class="QLineEdit" name="loginLineEdit" />
    </item>
    <item row="1" column="0" colspan="2" >
     <widget class="QLabel" name="label_2" >
      <property name="text" >
       <string>Wengo Password:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2" >
     <widget class="QLineEdit" name="passwordLineEdit" />
    </item>
    <item row="2" column="0" colspan="3" >
     <widget class="QPushButton" name="connectButton" >
      <property name="text" >
       <string>Connect to Wengo server / Send HTTP request</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="3" >
     <spacer>
      <property name="orientation" >
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" >
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="4" column="0" colspan="3" >
     <widget class="QTextBrowser" name="answerTextBrowser" />
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>274</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar" />
  <widget class="QToolBar" name="toolBar" >
   <property name="windowTitle" >
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea" >
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
